netci.groovy

```diff
@@ -276,6 +276,44 @@ def buildArchConfiguration = ['Debug': 'x86',
 }
 
 // **************************
+// Define outerloop testing for netfx that can build and run.  Run locally on each machine.
+// **************************
+[true, false].each { isPR ->
```

Comment:
We should probably make a couple more maps to help eliminate all the duplicated looping. We could add a mapp for targetgroup to OS and loop on just that set.